Abha_Rana/Tic-Tac-Toe/tt.css

```diff
@@ -0,0 +1,32 @@
+* {
+    box-sizing: border-box;
```

Comment:
Proper indentation required for every ruleset

Reply:
indentation is done via prettier.